executor/executor_test.go

```diff
@@ -5841,6 +5838,8 @@ func (s *testSuiteWithCliBaseCharset) TestCharsetFeature(c *C) {
 }
 
 func (s *testSuiteWithCliBaseCharset) TestCharsetFeatureCollation(c *C) {
+	collate.SetNewCollationEnabledForTest(true)
```

Comment:
Move it to testSuite?

Reply:
put it here is fine, just like the other tests

Reply:
Or use `testSuiteWithCliBase` instead of `testSuiteWithCliBaseCharset` and then remove `testSuiteWithCliBaseCharset`.